packages/@storefront/flux-capacitor/CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [Unreleased] [minor]
+### Added
+- SF-1073: Created new actions `selectMutlipleRefinements` and `selectMutliplePastPurchaseRefinements` actions.
```

Comment:
should be `selectMultipleRefinements` and `selectMultiplePastPurchaseRefinements` (typos). kind of important bc ppl might try to search these files

Reply:
My bad on the typos, I think I just glossed over them. Will be more diligent!